ice-cream-factory-datapoints-extractor/ice_cream_factory_datapoints_extractor/extractor.py

```diff
@@ -65,16 +67,17 @@ def run_extractor(
         config: Configuration parameters
         stop_event: Cancellation token, will be set when an interrupt signal is sent to the extractor process
     """
-
-    print("Starting Ice Cream Factory datapoints extractor")
+    logging.basicConfig(level=logging.INFO)
```

Comment:
Logging setup is done by extractor utils according to what is specified in the config file the user provided

Reply:
There's a provided `setup_logging` function:
https://github.com/cognitedata/python-extractor-utils/blob/master/cognite/extractorutils/configtools.py#L305
which is called during extractor startup here:
https://github.com/cognitedata/python-extractor-utils/blob/master/cognite/extractorutils/base.py#L262

This is preferred since it makes the output of logs in the same format as other extractors, and lets users configure logging to files etc.

Reply:
Okay. It was not clear that logging is magically configured deep in the stack

Reply:
Could very probably be better documented, I'll have a look and see if we can make it clearer!